src/ceph-disk/ceph_disk/main.py

```diff
@@ -5658,10 +5671,20 @@ def main(argv):
     CEPH_PREF_GROUP = args.setgroup
 
     if args.verbose:
-        args.func(args)
+        try:
+            args.func(args)
+        except Exception:
+            # warn on any exception when not running with verbosity
```

Comment:
is this backwards?  this looks like the verbose path

Reply:
Yes, my wording is backwards, hah!